### PR TITLE
Never change this valuegit diff! EVERgit diff!

### PR DESCRIPTION
**Pull request by mark-i-m**
Lol, That's what I get for including `!!` in a bash command.

Ping @Ayyko... What do you think?

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -33,9 +33,13 @@
 
 HOMEPAGE = "http://chaosthebot.com"
 
-# TEST SETTING PLEASE IGNORE
+# TEST SETTING
```
Comment (PlasmaPower) [inline settings.py:36]: The author of this forgot to add `TEST COMMENT PLEASE IGNORE`. Classic mistake.

Comment (mark-i-m) [reply settings.py:36]: No. No. No. You're suppose to pay very close attention to that variable. It's `TSPI` that you should ignore...

Comment (PlasmaPower) [reply settings.py:36]: What `TSPI`? I don't see anything :wink:

Comment (mark-i-m) [reply settings.py:36]: Look at line 41.

**Pull request closed without merging**